sky/clouds/service_catalog/images/aws_image_gen.py

```diff
@@ -0,0 +1,82 @@
+"""
+Run `python aws_image_gen.py` to copy the AMI in the source region to all regions and make them public.
+Will also generate a `images.csv` or append to an existing one; then we can refresh the `images.csv` inSkyPilot Catalog.
+"""
+
+import csv
+import json
+import os
+import subprocess
+import time
+
+# Inputs based on Packer generated AMI
+IMAGE_REGION = 'us-east-1'
+IMAGE_ID = 'ami-0981cc842c7188227'
+BASE_IMAGE_ID = 'ami-005fc0f236362e99f'
+OS_TYPE = 'ubuntu'
+OS_VERSION = '22.04'
+
+OUTPUT_CSV = 'images.csv'
+ALL_REGIONS = [
+    'us-west-1',
+]
+
+
+def copy_image_and_make_public(source_region, source_image_id, target_region):
+    # Copy the AMI to the target region
+    copy_command = f"aws ec2 copy-image --source-region {source_region} --source-image-id {source_image_id} --region {target_region} --name 'skypilot-aws-gpu-ubuntu-{time.time()}'"
```

Comment:
nit: break these into multiple lines to comply to our format requirement of line width <= 80 charactors

Reply:
Do we need to add `--output json` to make sure the output is in json format?